specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/securitydomain.json

```diff
@@ -0,0 +1,336 @@
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SecurityDomainCertificateItem"
+          },
+          "minItems": 3,
+          "maxItems": 3,
+          "uniqueItems": true,
+          "description": "Three certificates needed from customer"
+        }
+      },
+      "required": [
+        "certificates"
+      ]
+    },
+    "SecurityDomainCertificateItem": {
+      "properties": {
+        "value": {
+          "$ref": "#/definitions/SecurityDomainJsonWebKey",
+          "description": "Customer generated certificate containing public key in JWK format"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "TransferKey": {
+      "properties": {
+        "transfer_key": {
+          "$ref": "#/definitions/SecurityDomainJsonWebKey"
+        }
+      },
+      "required": [
+        "transfer_key"
+      ]
+    },
+    "SecurityDomainObject": {
+      "properties": {
+        "value": {
+          "type": "string"
+        }
+      }
+    },
+    "SecurityDomainOperationStatus": {
+      "properties": {
+        "status": {
+          "description": "operation status",
+          "enum": [
+            "Success",
+            "InProgress",
+            "Failed"
+          ],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "OperationStatus"
+          },
+          "type": "string"
+        },
+        "status_details": {
+          "type": "string"
+        }
+      }
+    },
+    "SecurityDomainJsonWebKey": {
+      "properties": {
+        "kid": {
+          "type": "string",
+          "description": "Key identifier."
+        },
+        "kty": {
+          "type": "string",
+          "description": "JsonWebKey Key Type (kty), as defined in https://tools.ietf.org/html/draft-ietf-jose-json-web-algorithms-40. For security domain this value must be RSA",
+          "enum": [
+            "RSA",
+            "RSA-HSM"
+          ],
+          "x-ms-enum": {
+            "name": "JsonWebKeyType",
+            "modelAsString": true
+          }
+        },
+        "key_ops": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "description": "Supported key operations."
+          }
+        },
+        "n": {
+          "type": "string",
+          "description": "RSA modulus."
+        },
+        "e": {
+          "type": "string",
+          "description": "RSA public exponent."
+        },
+        "x5c": {
+          "description": "X509 certificate chain parameter",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "use": {
+          "description": "Public Key Use Parameter. This is optional and if present must be enc.",
+          "type": "string"
+        },
+        "x5t": {
+          "description": "X509 certificate SHA1 thumbprint. This is optional.",
+          "type": "string"
+        },
+        "x5t#S256": {
```

Comment:
This value, however, is not in keys.json (at least, when I refactored it). This would need to be added.